review_july (#172)
fiskaltrust/productdescription-de-doc: product-service-description/compliance-as-a-service/features/basics/tse-as-a-service.md

> Als Hardware TSE kommt zur Zeit eine TSE von der Bundesdruckerei/Cryptovision zum Einsatz. Der TSE ist ein MicroSD-USB Adapter beigelegt. In einer fiskaltrust.Sorglos mit Hardware TSE-Subscription wird bei erstmaliger Bestellung die Hardware TSE inklusive USB Adapter an die in der Bestellung angegebene Lieferadresse geliefert. In jeder 12-monatigen Verlängerung dieser Subscription ist ein einmaliger Anspruch auf den Tausch bzw. Ersatz der Hardware TSE enthalten. 

Reviewer: ```suggestion
Als Bestandteil des Produktes "TSE-as-a-Service" mit einer Hardware-TSE kommt zur Zeit eine TSE der [Bundesdruckerei/Cryptovision](https://docs.fiskaltrust.cloud/docs/product-description/germany/products-and-services/caas/features/basics/tse/cryptovision#cryptovisionbundesdruckerei-interoperabilit%C3%A4t) zum Einsatz. Der TSE ist ein MicroSD-USB Adapter beigelegt. In einer fiskaltrust.Sorglos mit Hardware-TSE-Subscription wird bei erstmaliger Bestellung die Hardware-TSE inklusive USB Adapter an die Adresse des in der Bestellung ausgewählten Standorts (Outlet) geliefert. In jeder 12-monatigen Verlängerung dieser Subscription ist ein einmaliger Anspruch auf Tausch bzw. Ersatz der Hardware-TSE (z.B. im Falle eines Defekts, Ablaufs des Zertifikats oder des Erreichens der maximalen Signaturanzahl) enthalten. 
```

> Die zur Verfügung gestellte Hardware TSE bleibt im Eigentum der fiskaltrust und ist bei Tausch sowie bei Vertragsende an diese zurück zu senden.

Reviewer: ```suggestion
Die zur Verfügung gestellte Hardware-TSE bleibt im Eigentum der fiskaltrust und ist bei Tausch sowie bei Vertragsende an diese zurück zu senden.
```

> ### Cloud TSE-as-a-Service

Reviewer: ```suggestion
### Cloud-TSE-as-a-Service
```
